src/Services/Page.vala

```diff
@@ -78,6 +86,23 @@ public class ENotes.PageTable : DatabaseTable {
         }
 
         set_table_name ("Page");
+          
```

Comment:
Make sure to remove trailing whitespaces

Reply:
done